lib/config.js

```diff
@@ -407,8 +407,14 @@ Config.prototype.getConfig = function(filePath) {
     // Step 8: Merge in command line globals
     config = util.mergeConfigs(config, { globals: this.globals });
 
+    // Step 9: Merge in command line plugins
+    if (this.options.plugins) {
```

Comment:
I mean, I'd like to see a test in tests/lib/config.js for this. You added a test for CLI, but that only transitively tests the change. You should also test it directly.